modules/recommended-etcd-practices.adoc

```diff
@@ -18,6 +18,20 @@ For more information about defragmenting etcd, see the "Defragmenting etcd data"
 
 Because etcd writes data to disk and persists proposals on disk, its performance depends on disk performance. Slow disks and disk activity from other processes can cause long fsync latencies. Those latencies can cause etcd to miss heartbeats, not commit new proposals to the disk on time, and ultimately experience request timeouts and temporary leader loss. Run etcd on machines that are backed by SSD or NVMe disks with low latency and high throughput. Consider single-level cell (SLC) solid-state drives (SSDs), which provide 1 bit per memory cell, are durable and reliable, and are ideal for write-intensive workloads.
 
+The following hard disk features support optimal etcd performance:
+
+* Low latency to support fast read operation.
+* High-bandwidth writes for faster compactions and defragmentation.
+* High-bandwidth reads for faster recovery from failures.
+* Solid state drives as a minimum selection, however NVME drives are preferred.
+* Server-grade hardware from various manufacturers for increased reliability.
+* RAID 0 or RAID 1 configurations for reliable data.
+* Dedicated etcd drives for heavy input-output workloads, such as log files.
```

Comment:
the meaning I wanted to convey here is that customers should **not** co-locate the log files on the same drives that etcd uses. Can you rework that sentence to make it clear please?